### PR TITLE
chore: Add tests/etc

### PR DESCRIPTION
**Pull request by wstrange**
test scripts got missed as they were ignored by bin/ rule in .gitignore . 

Fixed by renaming tests/bin to tests/etc

### CODE REVIEW

File: tests/etc/auto-down.sh [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+ gcloud container clusters delete warren-ds-test --region us-west4
```
Comment (jrcast) [inline tests/etc/auto-down.sh:3]:
```suggestion
 gcloud container clusters delete NAME --region us-west4
```
Need a better name for the cluser? 🤷

File: tests/etc/auto-up.sh [hunk 2]
```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+gcloud beta container --project "engineering-devops" clusters create-auto "warren-ds-test" \
```
Comment (jrcast) [inline tests/etc/auto-up.sh:2]: same here

Comment (jrcast) [reply tests/etc/auto-up.sh:2]: Do we want to expose this here? Seems to be targeting our private gcp account.

Comment (wstrange) [reply tests/etc/auto-up.sh]: fixed

File: tests/etc/auto-up.sh [hunk 3]
```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+# Sample script to create an autopilot test cluster on GKE. Adjust for your environment.
+
+project=$(gcloud config list --format 'value(core.project)') 2>/dev/null
+
+
+gcloud beta container --project $project clusters create-auto "ds-test" \
+    --region "us-west4" --release-channel "regular" \
+    --network "projects/engineering-devops/global/networks/forgeops" \
+    --subnetwork "projects/engineering-devops/regions/us-west4/subnetworks/forgeops" \
```
Comment (jrcast) [inline tests/etc/auto-up.sh:10]: use $project here as well

File: tests/etc/auto-up.sh [hunk 4]
```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env bash
+# Sample script to create an autopilot test cluster on GKE. Adjust for your environment.
+
+project=$(gcloud config list --format 'value(core.project)') 2>/dev/null
+
+
+gcloud beta container --project $project clusters create-auto "ds-test" \
+    --region "us-west4" --release-channel "regular" \
+    --network "projects/engineering-devops/global/networks/forgeops" \
```
Comment (jrcast) [inline tests/etc/auto-up.sh:9]: +1

**Review by lee-baines — APPROVED** [review]

**Pull request merged**